src/test/stateful_modelinstance_test.cpp

```diff
@@ -165,6 +167,74 @@ class StatefulModelInstanceTest : public ::testing::Test {
     }
 };
 
+void StartStatefulPredict(const std::shared_ptr<ovms::ModelInstance> modelInstance, inputs_info_t modelInput, uint64_t seqId) {
+    tensorflow::serving::PredictRequest request = preparePredictRequest(modelInput);
+    setRequestSequenceId(&request, seqId);
+    setRequestSequenceControl(&request, SEQUENCE_START);
+
+    std::unique_ptr<ovms::ModelInstanceUnloadGuard> unload_guard;
+    tensorflow::serving::PredictResponse response;
+    // Do the inference
+    ASSERT_EQ(modelInstance->infer(&request, &response, unload_guard), ovms::StatusCode::OK);
+    // Check response
+    EXPECT_TRUE(CheckSequenceIdResponse(response, seqId));
+}
+
+void EndStatefulPredict(const std::shared_ptr<ovms::ModelInstance> modelInstance, inputs_info_t modelInput, uint64_t seqId) {
+    tensorflow::serving::PredictRequest request = preparePredictRequest(modelInput);
+    setRequestSequenceId(&request, seqId);
+    setRequestSequenceControl(&request, SEQUENCE_END);
+
+    std::unique_ptr<ovms::ModelInstanceUnloadGuard> unload_guard;
+    tensorflow::serving::PredictResponse response;
+    // Do the inference
+    ASSERT_EQ(modelInstance->infer(&request, &response, unload_guard), ovms::StatusCode::OK);
+    // Check response
+    EXPECT_TRUE(CheckSequenceIdResponse(response, seqId));
+}
```

Comment:
The only difference between those methods is sequence control input right? We expect successful `infer() `call and `sequence_id` in the response in both cases. Perhaps we could have one method for it with additional parameter - 
 `sequenceControlInput`